config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
I'll go with the first suggestion, since it is more usual to run the tests as

```console
$ bundle exec rake test
```

Rather than

```console
$ SECRET_BASE_BASE=964d510a3ddb702406b8226673b29f84a045eff2ce7640580973770f924cda54 bundle exec rake test
```

Ok?


Nobody answered my original question about having the dev and test secret in the repo :cry: 

I had looked at a few open source Rails projects looking for inspiration when I realized this setting needed to change, and apart from the obviously bad examples that store the production secret publicly, no code seemed to try to get a value from the secret from the environment for tests -- as it makes no difference AFAIK.

This is one example that looks sane:
https://github.com/RailsApps/rails-omniauth/blob/master/config/secrets.yml